cmds/core/init/init_unix.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !linux,!plan9
+
+// This is intended to work with Unix implementations,
```

Comment:
please remove this comment, it looks like a package comment and godoc will be confused.

Reply:
and if it doesn't work on bsd, let's remove this file and leave it for a day when someone will actually make it work?

Reply:
I wish we had working bsd but yeah ...

Reply:
removed.